addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js

```diff
@@ -19,31 +20,14 @@ FormRenderer.include({
      */
     async on_attach_callback() {
         this._super(...arguments);
-        // Useful when (re)loading view
-        if (this._hasChatter()) {
-            if (!this._chatterComponent) {
-                if (!this._chatterLocalId) {
-                    this._chatterLocalId = this.env.store.dispatch('createChatter', {
-                        initialThreadId: this.state.res_id,
-                        initialThreadModel: this.state.model,
-                    });
-                }
-                this._makeChatterComponent();
-            }
-            await this._mountChatterComponent();
-        }
+        WidgetAdapterMixin.on_attach_callback.call(this);
```

Comment:
The original `FormRenderer` should directly use the compatibility stuff instead (in web).